symfony.lock

```diff
@@ -5,6 +5,9 @@
     "barryvdh/elfinder-flysystem-driver": {
         "version": "v0.2.1"
     },
+    "becklyn/ordered-form-bundle": {
+        "version": "1.4.0"
+    },
```

Comment:
can you also update the project-base symfony.lock?

Reply:
I was thinking about it and then realized this should be done during the release, right? Do we need it beforehand? My point is, I am probably not able to do that 100% correctly manually, right?